src/main/java/org/apache/accumulo/testing/continuous/ContinuousInputFormat.java

```diff
@@ -143,8 +143,7 @@ private Key genKey(CRC32 cksum) {
 
         byte[] fam = genCol(random.nextInt(maxFam));
         byte[] qual = genCol(random.nextInt(maxQual));
-        @SuppressWarnings("deprecation")
-        byte[] cv = visibilities.get(random.nextInt(visibilities.size())).flatten();
+        byte[] cv = visibilities.get(random.nextInt(visibilities.size())).getExpression();
```

Comment:
`flatten` was used to normalize the existing parse tree. 
While calling `getExpression()` will  return a byte array, I don't believe it de-duplicates the expression anymore.

Just something to be aware of when testing.